controllers/comments-controller.js

```diff
@@ -14,4 +18,22 @@ function getCommentsByArticleId(request, response, next) {
     .catch((error) => next(error));
 }
 
-module.exports = { getCommentsByArticleId };
+function postNewComment(request, response, next) {
+  const { article_id } = request.params;
+  const { username, body } = request.body;
+
+  checkIfArticleExists(article_id)
```

Comment:
It's not necessary to check the article exists, if we get empty rows we know PSQL has not found an article with that Id